### DISCUSSION TITLE
Detect and underline URLs starting with www

### DISCUSSION THREAD

**Discussion by tommyrot** [kovidgoyal/kitty#7416] [Q&A] [answered]
How do i get kitty to detect and make clickable URLs like _www.example.com_ ? I've tried `url_prefixes www` but this option seems to require a scheme.

**Comment by kovidgoyal** [comment, marked as answer]
The scheme is the part before :// in the URL. www.whatever.com is not a URL. If you want to open it use the hints kitten with a regex to match it.
